mla/_models.py

```diff
@@ -571,7 +571,7 @@ def inject_background_events(self) -> np.ndarray:
         n_background_observed = np.random.poisson(self._n_background)
         background = np.random.choice(self._data, n_background_observed).copy()
         background['ra'] = np.random.uniform(0, 2 * np.pi, len(background))
-        background['time'] = self._background_time_profile.random(
+        background['time'] = self.background_time_profile.random(
```

Comment:
Why the scramble time part is not in here , replacing this line? You are doing time sampling twice here.

Reply:
fixed